main/src/main/java/cgeo/geocaching/maps/google/v2/GoogleMapView.java

```diff
@@ -425,24 +433,13 @@ public boolean onSingleTapUp(final MotionEvent e) {
                         final Point waypointPoint = googleMap.getProjection().toScreenLocation(new LatLng(closest.getCoord().getCoords().getLatitude(), closest.getCoord().getCoords().getLongitude()));
                         if (insideCachePointDrawable(p, waypointPoint, closest.getMarker(0).getDrawable())) {
                             onCacheTapListener.onCacheTap(closest.getCoord());
```

Comment:
Works fine! Maybe move the cache tap listener as well to fix #7569?

Reply:
Thanks for reviewing.

I had tried to move `cacheTapListener` to `mapOnClickListener` as well, but that will trigger the enormous delay when tapping on a cache marker, which (according to the comment in the source) probably was the reason to write the `GestureListener` in the first place. Thus I reverted to stay with triggering action bar hiding/unhiding.